packages/components/icon/stories/icon.stories.js

```diff
@@ -0,0 +1,8 @@
+import { storiesOf } from '@storybook/html';
+import '@tradeshift/elements';
+import '@tradeshift/elements.icon';
+
+storiesOf('ts-icon', module).add('type=""', () => {
+	const icon = document.createElement('ts-icon');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "icon". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS1488)